lemmy_apub/src/inbox/community_inbox.rs

```diff
@@ -56,6 +65,12 @@ pub async fn community_inbox(
     .actor()?
     .as_single_xsd_any_uri()
     .context(location_info!())?;
+  debug!(
+    "Community {} inbox received activity {:?} from {}",
+    community.name,
+    &activity.id_unchecked(),
+    &user_uri
+  );
```

Comment:
This kind of stuff could easily be `info!()`.

Reply:
mmk , I'll change.